package.json

```diff
@@ -37,7 +37,7 @@
   },
   "devDependencies": {
     "@cryptape/sdk-ts-config": "0.0.1",
-    "@types/jest": "^24.0.0",
+    "@types/jest": "^23.0.0",
```

Comment:
If the neuron-ui uses jest@23 and neuron-wallet uses jest@24, the @types/jest is better to be added in them, respectively.

Reply:
They use jest@23 together now.